internal/convert/api_keys_access_list.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2021 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package convert
+
+import atlas "go.mongodb.org/atlas/mongodbatlas"
+
+func FromWhitelistAPIKeysToAccessListAPIKeys(in *atlas.WhitelistAPIKeys) *atlas.AccessListAPIKeys {
```

Comment:
Since this and `FromAccessListAPIKeysReqToWhitelistAPIKeysReq` are public can you please document them? also probably with a note about om 4.2 and 4.4?